### PR TITLE
Linsen chose to change their api D:

### PR DESCRIPTION
**Pull request by The1Penguin**
Please fix

### CODE REVIEW

**Review by Multipacker — APPROVED** [review]
LGTM

**Comment by SwedishSubmarine** [comment]
Add comment about what magic numbers mean

File: src/Model/Linsen.hs [hunk 1]
```diff
@@ -25,8 +25,8 @@ import           Data.List.Extra                          ( (!?) )
 import           Data.Text.Lazy                           ( Text
                                                           , replace
                                                           , strip )
-import           Data.Thyme                               ( parseTime
-                                                          , defaultTimeLocale )
+import           Data.Thyme                               ( parseTime )
+import           Data.Time.Format                         (TimeLocale (..) )
```
Comment (Rembane) [inline src/Model/Linsen.hs]: Can you use https://hackage.haskell.org/package/thyme-0.4/docs/Data-Thyme-Format.html#t:TimeLocale instead? So you don't have to add another dependency.

File: test/Main.hs [hunk 2]
```diff
@@ -84,16 +84,16 @@ main = hspec $ do
       s1 <- BL.readFile "test/linsen1.json"
       testFun
         (Right [ Menu
```
Comment (Rembane) [inline test/Main.hs:86]: I love the updating of the example.

File: src/Model/Linsen.hs [hunk 3]
```diff
@@ -92,9 +128,9 @@ parse day =
 
   menuParser :: [Value] -> Parser [Menu]
   menuParser = pure . (zip [0 :: Integer ..] >=> \case
```
Comment (Rembane) [inline src/Model/Linsen.hs:136]: Magic indices <3

These are hard to make less magic. :(

**Review by Rembane — CHANGES_REQUESTED** [review]
Good stuff! See requests in comments! Keep up the good work!

File: test/Main.hs (hunk 2 above)
Comment (Rembane) [reply test/Main.hs:86]: Maybe I should eat lunch.

File: src/Model/Linsen.hs (hunk 1 above)
Comment (The1Penguin) [reply src/Model/Linsen.hs]: So, about that

Comment (The1Penguin) [reply src/Model/Linsen.hs]: You see, Thyme uses Time as the dependency, which this constructor, but yes, it does reexport it so I will change that

Comment (Rembane) [reply src/Model/Linsen.hs]: Sweet!

File: src/Model/Linsen.hs (hunk 3 above)
Comment (Rembane) [reply src/Model/Linsen.hs:136]: I was wrong! :D

File: mat-chalmers.cabal [hunk 4]
```diff
@@ -55,7 +55,6 @@ library
                , word8 == 0.1.3
                , extra >= 1.7.16 && <= 1.8
                , vector-space >= 0.16 && <0.18
-               , time >= 1.11 && <1.20
```
Comment (Rembane) [inline mat-chalmers.cabal]: \o/

**Review by Rembane — APPROVED** [review]
This looks good!

File: src/Model/Linsen.hs [hunk 5]
```diff
@@ -37,6 +38,44 @@ import           Model.Types                              ( NoMenu(..)
...
+        , dateFmt = "%d/%m/%y"
+        , timeFmt = "%H:%M:%S"
+        , time12Fmt = "%I:%M:%S %p"
+        , knownTimeZones = []
+        }
+
+pattern MeatDish, FishDish, VegDish :: Integer
+pattern MeatDish = 3
+pattern FishDish = 8
+pattern VegDish = 13
```
Comment (SwedishSubmarine) [inline src/Model/Linsen.hs:77]: Vad är oddsen att de kommer ändra dessa igen...

**Review by SwedishSubmarine — APPROVED** [review]

File: src/Model/Linsen.hs (hunk 5 above)
Comment (The1Penguin) [reply src/Model/Linsen.hs:77]: T-T

**Pull request merged**